test/e2e/oci/oci_test.go

```diff
@@ -109,6 +109,7 @@ func TestEnablePrintStatementsForBundles(t *testing.T) {
 		if !strings.Contains(buf.String(), expContains) {
 			t.Fatalf("expected logs to contain %q but got: %v", expContains, buf.String())
 		}
+		time.Sleep(3 * time.Second) // wait a couple of seconds for the second trigger to kick in
```

Comment:
Seems to me like this should be included in a separate PR.

Reply:
:+1: spinning this out into an extra PR is a good idea!

Reply:
👍

Reply:
Extracted into https://github.com/open-policy-agent/opa/pull/4806